orion-server/src/main/java/com/pinterest/orion/core/automation/operator/kafka/BrokerHealingOperator.java

```diff
@@ -54,7 +52,7 @@ public class BrokerHealingOperator extends KafkaOperator {
   private long maxNumStaleIntervals = 2; // default 2 times
   private double unhealthyBrokerURPRatioThreshold = 0; // default no URPs allowed
   private int unhealthyAlertFailThreshold = 3;
```

Comment:
Was this not converted to a config?

Reply:
These configs are not part of my change. They are the thresholds for alerting. 

They are already configurable. You can check line 46-50 and initialize method.